app-text/apvlv/apvlv-0.4.0.ebuild

```
@@ -0,0 +1,27 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit xdg cmake
```

Empty line between please.
```suggestion
EAPI=8

inherit xdg cmake
```